libfwupdplugin/fu-device.c

```diff
@@ -5519,6 +5534,8 @@ fu_device_ensure_from_component(FuDevice *self, XbNode *component)
 		fu_device_ensure_from_component_signed(self, component);
 	if (fu_device_has_internal_flag(self, FU_DEVICE_INTERNAL_FLAG_MD_SET_VERFMT))
 		fu_device_ensure_from_component_verfmt(self, component);
+	if (fu_device_has_internal_flag(self, FU_DEVICE_INTERNAL_FLAG_MD_SET_FLAGS))
```

Comment:
I guess you're missing to add this flag to NVME plugin still

Reply:
Yup, will do in the other PR.